Patch to fix missing link libraries in pkgconfig
for correctly linking with external Lua.
author: Arnold H. Kole (Utrecht University)
diff --git a/flook.pc.in b/flook.pc.in
index 680b5e0..b46f8ad 100644
--- a/flook.pc.in
+++ b/flook.pc.in
@@ -10,5 +10,5 @@ Version: @PROJECT_VERSION@
 URL: https://github.com/ElectronicStructureLibrary/flook
 Cflags: -I${includedir}
 Libs.private: -ldl
-Libs: -L${libdir} -lflookall
+Libs: -L${libdir} -lflookall -llua -ldl
 
